hack/run-test.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+readonly CURRENT_SCRIPT=$(basename $0)
+
+HEADER_MESSAGE="Running Tests "
+
+usage() {
+    print "Usage:"
+    print "  ${CURRENT_SCRIPT} [-h] -s suites -p extraparams [-m header_message]"
+    print ""
+    print "Options:"
+    print "    -h                Help for ${CURRENT_SCRIPT}"
+    print "    -t                list of space separated paths to Testsuites to execute"
+    print "    -p                string with extra Params for ginkgo"
+    print "    -m                Message header to print before test execution. Default '${HEADER_MESSAGE}'"
+}
+
+exit_error() {
+    print ""
+    print "error: $*"
+    exit 1
+}
+
+print() {
+    echo "$*" >&2
```

Comment:
`print`ing on stderr is a bit misleading, this is more akin to a log than a regular print (usually done on stdout)

Reply:
You're right.
Changed.